mindjunkies/forums/views.py

```
@@ -1,129 +1,189 @@
-            "forums/partials/like_button.html",
-            context=self.get_context_data(course=topic.course, topic=topic),
-        )
\ No newline at end of file
+class ReplyFormView(LoginRequiredMixin, CourseContextMixin,View):
+    def get(self, request, *args, **kwargs):
+        reply_id = self.kwargs.get("reply_id")
+        print(f"Reply ID received: {reply_id}")
+        reply = get_object_or_404(Reply, id=reply_id)
+        if not reply:
+            reply=get_object_or_404(ForumComment,id=reply_id)
+        replyForm = ForumReplyForm()
```

Ruff (0.8.2)

Variable `replyForm` in function should be lowercase

(N806)